### PR TITLE
Update known issue 517815-BCL Stack traces are missing source information

### PR DESCRIPTION
**Pull request by noahfalk**
Add the portable PDB conversion tool as a workaround now that it is better documented how to use it.

### CODE REVIEW

**Comment by noahfalk** [comment]
@vivmishra @rpetrusha @tmat

**Comment by tmat** [comment]
👍

File: releases/net471/KnownIssues/517815-BCL Stack traces are missing source information for frames with debug information in the Portable PDB format.md [hunk 1]
```diff
@@ -12,7 +13,8 @@ The .NET Framework 4.7.1 added support for detecting and parsing the Portable PD
 ## Workarounds
 
 - If you control the build process for the problematic assemblies you may be able to configure it to generate the classic Windows PDB format instead.
+- You could use the [PDB conversion tool](https://github.com/dotnet/symreader-converter) to convert the Portable PDBs into the classic Windows PDB format and deploy those with the application instead.
 
 ## Resolution
 
-A fix is anticipated in .NET Framework 4.7.2, in the near future, that restores Portable PDB functionality with greatly improved performance.
\ No newline at end of file
+A fix is anticipated in .NET Framework 4.7.2, in the near future, that restores Portable PDB functionality with greatly improved performance.
```
Comment (rpetrusha) [inline releases/net471/KnownIssues/517815-BCL Stack traces are missing source information for frames with debug information in the Portable PDB format.md]: the commas after 4.7.2 and future should be removed

File: releases/net471/KnownIssues/517815-BCL Stack traces are missing source information for frames with debug information in the Portable PDB format.md [hunk 2]
```diff
@@ -12,7 +13,8 @@ The .NET Framework 4.7.1 added support for detecting and parsing the Portable PD
 ## Workarounds
 
 - If you control the build process for the problematic assemblies you may be able to configure it to generate the classic Windows PDB format instead.
+- You could use the [PDB conversion tool](https://github.com/dotnet/symreader-converter) to convert the Portable PDBs into the classic Windows PDB format and deploy those with the application instead.
```
Comment (rpetrusha) [inline releases/net471/KnownIssues/517815-BCL Stack traces are missing source information for frames with debug information in the Portable PDB format.md]: could --> can (can is less hypothetical)

**Comment by noahfalk** [comment]
Thanks @rpetrusha !

**Comment by rpetrusha** [comment]
Thanks for making the changes, @noahfalk. I'll merge this now.

**Pull request merged**